### PR TITLE
UCS/TEST: Fix segfault in UCP, due to int truncation in count_one_bits()

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8425/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8431/ for details (Mellanox internal link).

File: src/ucs/arch/bitops.h [hunk 1]
```diff
@@ -94,15 +94,17 @@
 )
 
 /* Returns the number of 1-bits in x */
-#define ucs_count_one_bits(x)      __builtin_popcount(x)
+#define ucs_popcount(x) \
```
Comment (brminich) [inline src/ucs/arch/bitops.h]: minor: would need to use `_` prefix in macro names

Comment (yosefe) [reply src/ucs/arch/bitops.h]: yeah, i didn't want to fix this issue too in this PR..

Comment (yosefe) [reply src/ucs/arch/bitops.h]: @brminich fixed

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/8437/ for details (Mellanox internal link).

**Pull request merged**